### PR TITLE
Skeleton holder visual update

### PR DESCRIPTION
**Pull request by eldiegod**
fixes: #941

### CODE REVIEW

**Review by kdembler — CHANGES_REQUESTED** [review]
There seems to be some issue in the Video Tile, with the bottom 2 placeholders. It seems like only one vertical half of them is animated:

https://user-images.githubusercontent.com/12646744/128160143-26f1562e-22b0-4a44-adce-1d95200ea662.mov

**Comment by eldiegod** [comment]
Avatar on channel view doesnt show wave also on studio view my uploads circle to the right doesn't show a wave

Also remove pulsing anim on the hero video from homepage

File: src/components/CoverVideo/CoverVideo.style.ts [hunk 1]
```diff
@@ -69,22 +69,6 @@ export const PlayerContainer = styled.div`
   ${absoluteMediaCss};
 `
 
-const pulse = keyframes`
```
Comment (kdembler) [inline src/components/CoverVideo/CoverVideo.style.ts]: Unused imports left on top of this file

**Review by kdembler — COMMENTED** [review]
Needs cleanup, otherwise ready to go

**Review by kdembler — APPROVED** [review]

**Pull request merged**